packages/darklang/stdlib/list.dark

```diff
@@ -80,6 +80,13 @@ module Darklang =
             PACKAGE.Darklang.Stdlib.List.``member`` tail value
 
 
+      let repeatUnsafe (times: Int) (value: 'a) : List<'a> =
```

Comment:
Can this be a lambda inside `repeat`? That way it's not accessible to other functions/users